contracts/src/v0.8/ccip/test/pools/BurnFromMint/BurnFromMintTokenPool.lockOrBurn.t.sol

```diff
@@ -1,37 +1,23 @@
 // SPDX-License-Identifier: BUSL-1.1
 pragma solidity 0.8.24;
 
-import {Pool} from "../../libraries/Pool.sol";
-import {RateLimiter} from "../../libraries/RateLimiter.sol";
-import {BurnFromMintTokenPool} from "../../pools/BurnFromMintTokenPool.sol";
-import {TokenPool} from "../../pools/TokenPool.sol";
-import {BurnMintSetup} from "./BurnMintSetup.t.sol";
+import {Pool} from "../../../libraries/Pool.sol";
+import {RateLimiter} from "../../../libraries/RateLimiter.sol";
+import {TokenPool} from "../../../pools/TokenPool.sol";
+import {BurnFromMintTokenPoolSetup} from "./BurnFromMintTokenPoolSetup.t.sol";
 
-import {IERC20} from "../../../vendor/openzeppelin-solidity/v4.8.3/contracts/token/ERC20/IERC20.sol";
-
-contract BurnFromMintTokenPoolSetup is BurnMintSetup {
-  BurnFromMintTokenPool internal s_pool;
-
-  function setUp() public virtual override {
-    BurnMintSetup.setUp();
-
-    s_pool = new BurnFromMintTokenPool(s_burnMintERC677, new address[](0), address(s_mockRMN), address(s_sourceRouter));
-    s_burnMintERC677.grantMintAndBurnRoles(address(s_pool));
-
-    _applyChainUpdates(address(s_pool));
-  }
-}
+import {IERC20} from "../../../../vendor/openzeppelin-solidity/v4.8.3/contracts/token/ERC20/IERC20.sol";
 
 contract BurnFromMintTokenPool_lockOrBurn is BurnFromMintTokenPoolSetup {
-  function test_Setup_Success() public view {
+  function test_setup_Success() public view {
     assertEq(address(s_burnMintERC677), address(s_pool.getToken()));
     assertEq(address(s_mockRMN), s_pool.getRmnProxy());
     assertEq(false, s_pool.getAllowListEnabled());
     assertEq(type(uint256).max, s_burnMintERC677.allowance(address(s_pool), address(s_pool)));
     assertEq("BurnFromMintTokenPool 1.5.0", s_pool.typeAndVersion());
   }
 
-  function test_PoolBurn_Success() public {
```

Comment:
Why did you change the casing on these? The test rename comes after and these new names are not following the final naming scheme anyway?

Reply:
it was giving annoying linting errors about the case which is why but I will revert it.

Reply:
no idea.